src/Jadu/Pulsar/Twig/Extension/HelperOptionsModifierExtension.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Jadu\Pulsar\Twig\Extension;
+
+class HelperOptionsModifierExtension extends \Twig_Extension
+{
+	/**
```

Comment:
Mix of hard tabs and spaces in this file, can you make sure your editor is reading the .editorconfig file?

http://editorconfig.org/#download